superduper/misc/eager.py

```diff
@@ -316,8 +317,8 @@ def _apply_node(self, node: SuperDuperData):
         listener = node.model.to_listener(
             key=key,
             select=select,
-            predict_id=predict_id,
-            identifier=predict_id,
+            identifier=predict_id.split('__')[0],
+            uuid=predict_id.split('__')[1],
             predict_kwargs=predict_kwargs,
```

Comment:
We just need to use the `identifier = self.model.identifier` here, because the because the UUID is generated automatically.